scripts/alert-missing-dumps.sh

```
@@ -1,49 +1,121 @@
+    -d "{\"source\":\"dump-alert\",\"entity\":\"dump-alert\",\"alert_type\":\"$1\",\"title\":\"$2\",\"message\":\"$3\",\"ts\":\"$(date -u +%Y-%m-%dT%H:%M:%SZ)\"}" \
+    >/dev/null 2>&1 || true
+}
+
+check_age() {
+  name="$1"; sev="$2"
+  ts=$(newest_ts "$name")
+  age=$(age_hours "$ts")
+  if [ "${age:-999999}" -gt "$MAX_AGE_HOURS" ]; then
+    MISSING=$((MISSING + 1))
+    echo "[$(date -u +%Y%m%d_%H%M%S)] ALERTA ${name}: ultimo='${ts:-NENHUM}' idade=${age}h (limite ${MAX_AGE_HOURS}h)"
+    touch "${STATE_DIR}/ALERT_DUMP_MISSING_${name//\//_}_${TODAY}" 2>/dev/null
```

Shellcheck (0.11.0)

[warning] In POSIX sh, string replacement is undefined.

(SC3060)